simtools/schemas/model_parameters/correct_nsb_spectrum_to_telescope_altitude.schema.yml

```diff
@@ -16,7 +16,7 @@ data:
     unit: dimensionless
     default: "atm_trans_2200_1_3_0_0_0.dat"
 instrument:
-  class: Telescope
+  class: Camera
```

Comment:
Suggest to leave it as `Telescopes`. Requires merging of #1173

Reply:
Done. Will wait to see that tests pass now.